assets/src/edit-story/elements/text/karma/edit.karma.js

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
[nit]: Will tests be added to a 'karma' folder or the 'test' / 'test/karma' folder?

Reply:
Haven't decided yet. Depends on how different they will end up. We might just call them `test/*.karma.js`. But if they get to be too different, I think it'd make sense to separate. Then either `karma/` or `test/karma` is fine.

Reply:
For now they will have to go to the `karma/` folder. It's just easier to completely separate them from Jest tests across linters and test runners.